src/server/database/migrations/20171115161524-fixRemovedTeamMember.js

```diff
@@ -0,0 +1,10 @@
+exports.up = async (r) => {
+  await r.table('Project')
+    .update((project) => ({
+      userId: project('teamMemberId').split('::')(0)
```

Comment:
I'm seeing a lot of code touching `teamMemberId`s as strings; a lot of split and join operations.

To be safe, it'd be best to abstract over the representation of these IDs, and define some functions to read and construct 'em as needed.

Reply:
that true, we should definitely write a helper kinda like relays `fromGlobalId`. However, it do it for this specific line, we'd need a second helper to do that within rethinkdb.

Reply:
Yeah, fair point! Though at least we can do this in app code!